frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -85,10 +86,12 @@ function testCore(actionType, editMode) {
     expect(addTask).toHaveBeenCalledTimes(0);
     expect(addEmail).toHaveBeenCalledTimes(1);
     expect(updateEmail).toHaveBeenCalledTimes(0);
+    expect(updateTask).toHaveBeenCalledTimes(0);
   } else if (actionType === ACTION_TYPE.email && editMode === EDIT_MODE.update) {
     expect(addTask).toHaveBeenCalledTimes(0);
     expect(addEmail).toHaveBeenCalledTimes(0);
     expect(updateEmail).toHaveBeenCalledTimes(1);
+    expect(updateTask).toHaveBeenCalledTimes(0);
   } else if (actionType === ACTION_TYPE.task && editMode === EDIT_MODE.create) {
     expect(addTask).toHaveBeenCalledTimes(1);
     expect(addEmail).toHaveBeenCalledTimes(0);
```

Comment:
Just missing one more " expect(updateTask).toHaveBeenCalledTimes(0);" in this clause

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/198/commits/5189b58d8309def83f374a7b60b4091b4063d01c